### ISSUE TITLE
Multiple frontendes for one container (service)

### ISSUE THREAD

**Issue by EugenMayer** [rawmind0/rancher-traefik#75]
I am aware of the discussion here https://github.com/rawmind0/rancher-traefik/issues/60

In my case, i usually have one container one port. But i have some exceptions which i cannot influence, one is Sonatype Nexus. It exposes each repo using a own port. So you might have maven on X, docker1 on Y, docker2 on Z, gem on Y

This makes it mandatory to define several frontends with fairly basic rules

So basically a set of backend / frontend rules per port, with a domain each. As far as i understand this should be perfectly possible with Traefik in general, it just needs multiple backends.

Is that a general issue since using rancher-metadata does not allow exposing several backends per service?

**Comment by rawmind0** [comment]
As told at #60, i get your point but it's a issue about traefik support it or not. 

I mean, traefik reads services labels from distinct backends (rancher, consul,....), and based on [templates](https://github.com/containous/traefik/tree/master/templates), generates and apply traefik config. Traefik doesn't support multiple port at any backend but docker, and PR to do it should be submitted there. :)

**Comment by EugenMayer** [comment]
Well for traffic, i am not talking about multiple ports on one backend, rather multiple backends on the same container with multiple frontends - thats something which is supported very much right.

But defining several backens + fronends on one container in rancher metadata is the issue then.. we would need a far more complex and "namespaced" label structure - as actually the namespaces are in traefik.
